templates/repo/issue/view_content/comments.tmpl

```diff
@@ -22,6 +22,7 @@
 								{{end}}
 							</div>
 						{{end}}
+						{{template "repo/issue/view_content/reactions" Dict "ctx" $ "ActionURL" (Printf "%s/issues/%d/comment/%d/reactions" $.RepoLink $.Issue.Index .ID) "Reactions" .GetReactionsByType }}
```

Comment:
Why can't we just do `{{template "repo/issue/view_content/reactions" .}}` like we do elsewhere?

Reply:
@ethantkoenig because this template needs to be used in loop and loop variable needs to be passed to that template

Reply:
Oh I see. On another note, there aren't handlers for the `:owner/:repo/issues/:index/comments/:id/reactions` route, so trying to add a reaction to a comment always results in a 404.